app/_hub/kong-inc/ai-prompt-decorator/overview/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+nav_title: Overview
+---
+
+The AI Prompt Decorator plugin adds an array of `llm/v1/chat` messages to either the start or end of a caller's chat history.
+This lets the caller create more complex prompts.
+
+You can use this plugin to set up specific prompt history, words, phrases, or otherwise have more control over how an LLM service is used when called via {{site.base_gateway}}.
```

Comment:
I'm still kind of confused about how this plugin works and what it can do for me. I'm not sure how prompt history would relate to Kong Gateway.